tests/nn/data_parallel/test_sharded_ddp.py

```diff
@@ -44,6 +44,7 @@ def check(broadcast_buffers: bool, grad_accumulation: bool = False) -> None:
         model.register_buffer("test_buffer", torch.ones((1)) * rank)
         model.to(device)
 
+        next(model.parameters()).requires_grad = False
```

Comment:
add a comment?

Reply:
Yes, I was about to right that this is the unit test change which makes sure that this is caught in the future. Will do !